src/components/HeroPicture.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+function HeroPicture() {
+  return (
+    <div className="hero-picture">
+    
+    <img src="./statue.png" alt="Generic problematic statue"></img>
```

Comment:
Better to import the image in JS, or use an absolute path, if this component was used on another page the src would look relative to that page and would 404